ipaserver/install/installutils.py

```diff
@@ -71,15 +71,19 @@
 class BadHostError(Exception):
     pass
 
+
```

Comment:
Please don't reformat unrelated code lines. This is causing additional noise and make backports harder.

Remove the additional new lines.

Reply:
done, i've discard all chages of PEP8 that I was making. fixing just in the lines I've changed.